packages/docsify-server-renderer/index.js

```diff
@@ -1,43 +1,34 @@
 import { readFileSync } from 'fs';
-import { resolve, basename } from 'path';
-import resolvePathname from 'resolve-pathname';
-import fetch from 'node-fetch';
-import debug from 'debug';
-import DOMPurify from 'dompurify';
+import { resolve as resolvePath, isAbsolute, basename } from 'path';
+import { resolve as resolveUrl } from 'url';
 import { AbstractHistory } from '../../src/core/router/history/abstract';
 import { Compiler } from '../../src/core/render/compiler';
-import { isAbsolutePath } from '../../src/core/router/util';
+import configDealer from '../../src/core/config';
```

Comment:
Can we rename it to `configHandler` or something similar

Reply:
Changed to `configHandler`.